host/class/uvc/usb_host_uvc/src/descriptor.c

```diff
@@ -190,8 +190,8 @@ int parse_configuration(libusb_config_descriptor_t *config, const uint8_t *buffe
             LIBUSB_GOTO_ON_FALSE(interface->altsetting);
 
             for (int ep = 0; ep < endpoints; ep++) {
-                ep_desc = usb_parse_endpoint_descriptor_by_index(ifc_desc, ep, config->wTotalLength, &offset);
-                ifc_desc = (const usb_intf_desc_t *)ep_desc;
+                int intf_offset = offset; // Current offset is interface offset in the configuration descriptor
```

Comment:
Is there any issue related? 
How I could verify changes?

Reply:
You can run uvc example in esp-idf and plug in ie. a MSC flash disk. Before the example crashed on NULL pointer access, after in correctly handles the error